lib/synapse.rb

```diff
@@ -33,6 +34,8 @@ def initialize(opts={})
       # configuration is initially enabled to configure on first loop
       @config_updated = true
 
+      @rate_limiter = RateLimiter.new(opts['rate_limiter_path'])
```

Comment:
We currently log from initialize() if the state file can't be read.  I think this is sufficient?